### PR TITLE
added waves and changed background

### PR DESCRIPTION
**Pull request by rvaidun**
Problem
=======
What problem or issue does this pull request solve [Jira](https://cruzhacks-dev.atlassian.net/jira/software/c/projects/C2D/issues/)
Added waves to 404 page
Changed background to match figma

Solution
========
What I/we did to solve this problem
* 

Change Summary:
---------------
* Tidy, well formulated commit message
* Another great commit message
* Something else I/we did

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* Insert any notes/issues, dependencies added or removed, or images

### CODE REVIEW

File: src/views/Error/index.view.tsx [hunk 1]
```diff
@@ -11,7 +14,15 @@ const ErrorView: React.FC = () => {
   }
   return (
     <div className='error-view'>
-      <div>
+      <div className='error-view__wavesdiv'>
```
Comment (krishnakpandian) [inline src/views/Error/index.view.tsx]: Can avoid including `div` in  the className

File: src/views/Error/index.view.tsx [hunk 2]
```diff
@@ -11,7 +14,15 @@ const ErrorView: React.FC = () => {
...
+        <img src={Waves} alt='waves' className='error-view__waves' />
+        <img
+          src={BiggerWaves}
+          alt='wavesbig'
+          className='error-view__wavesbig'
+        />
+      </div>
+      <div className='error-view__messagediv'>
```
Comment (krishnakpandian) [inline src/views/Error/index.view.tsx]: Same as before rename to avoid naming container with `div`

File: src/views/Error/index.view.tsx [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import * as React from "react"
 import "./index.scss"
 import { useHistory } from "react-router-dom"
+import Waves from "images/error/waves.svg"
+import BiggerWaves from "images/error/waves_dropshadow.svg"
```
Comment (krishnakpandian) [inline src/views/Error/index.view.tsx]: Can we try importing these as ReactComponents instead of images

**Review by krishnakpandian — CHANGES_REQUESTED** [review]
This looks really good! Just a couple of quick changes!

**Comment by sidhantas** [comment]
I'm not sure if this is just me, but the bottom wave on the error page looks kind of pixelated, it might b the shadow but here's a screenshot
image

**Comment by krishnakpandian** [comment]
> I'm not sure if this is just me, but the bottom wave on the error page looks kind of pixelated, it might b the shadow but here's a screenshot image

I definitely am now able to see it on the big wave, it might be from how it is scaled from the svg, I think we should try cleaning that up as well if possible

**Review by krishnakpandian — APPROVED** [review]

**Pull request merged**